charts/seed-controlplane/charts/kube-apiserver/templates/_helpers.tpl

```diff
@@ -90,3 +90,24 @@ rules:
 - level: None
 {{- end -}}
 {{- end -}}
+
+{{- define "kube-apiserver.serviceAccountConfig" -}}
+{{- if .Values.serviceAccountConfig }}
+{{- if .Values.serviceAccountConfig.issuer }}
```

Comment:
Can you also add all these fields in the `values.yaml` of this chart (commented), please?